libs/cart/testing/src/in-memory-backend/cart-root.service.ts

```diff
@@ -27,10 +27,12 @@ export class DaffInMemoryBackendCartRootService implements InMemoryDbService {
 
   constructor(
     private cartService: DaffInMemoryBackendCartService,
-    private cartItemsService: DaffInMemoryBackendCartItemsService,
+		private cartItemsService: DaffInMemoryBackendCartItemsService,
+		private storageService: DaffCartStorageService
   ) {}
 
   createDb(reqInfo: RequestInfo) {
+		this.storageService.removeCartId();
```

Comment:
This will throw an error in SSR. Not sure if that is a problem here, just wanted to make sure that you have considered it and will handle the error if appropriate.

Reply:
yea, you're right, there's definitely issues with this. Going to change

Reply:
completely changed how it works